pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -250,6 +250,24 @@ def test_agg_multiple_functions_maintain_order(df):
     tm.assert_index_equal(result.columns, exp_cols)
 
 
+def test_agg_multiple_functions_same_name(df):
+    # GH 30880
+    np.random.seed(1)
```

Comment:
why is this needed?

Reply:
No longer needed, sorry, came from a previous commit